libs/ui-watt/src/lib/components/text-field/watt-text-field.stories.ts

```diff
@@ -0,0 +1,163 @@
+/**
+ * @license
+ * Copyright 2020 Energinet DataHub A/S
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License2");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { Meta, moduleMetadata, StoryFn } from '@storybook/angular';
+import { FormControl, ReactiveFormsModule } from '@angular/forms';
+
+import { WattIconComponent } from '../../foundations/icon/icon.component';
+import { WattTextFieldComponent } from './watt-text-field.component';
+import { WattFieldComponent } from '../field';
+import { WattButtonComponent } from '../button';
+
+const meta: Meta<WattTextFieldComponent> = {
+  title: 'Components/Text Field',
+  component: WattTextFieldComponent,
+  decorators: [
+    moduleMetadata({
+      imports: [
+        ReactiveFormsModule,
+        WattTextFieldComponent,
+        WattIconComponent,
+        WattFieldComponent,
+        WattButtonComponent,
+      ],
+    }),
+  ],
+};
+
+export default meta;
+
+const template = `<div style="display: flex; gap: var(--watt-space-m); flex-direction: column; margin-bottom: var(--watt-space-m);">
+  <watt-field [label]="label">
+    <watt-text-field [required]="required" [type]="type" [placeholder]="placeholder" [formControl]="exampleFormControl"></watt-text-field>
+  </watt-field>
+</div><p>Value: {{exampleFormControl.value}}</p>`;
+
+const howToUseGuideBasic = `
+How to use
+
+1. Import ${WattTextFieldComponent.name} in a module or component
+
```

Comment:
suggestion: let's not promote modules anymore :)


```suggestion
1. Import ${WattTextFieldComponent.name} in a component

```